index.js

```diff
@@ -35,10 +30,23 @@ const contacts = [
   "Will Faurot"
 ]
 
-function listContacts() {
+$(document).ready(function(){
+  listContacts(contacts)
+  sortByFirstName()
+})
+
+
+function listContacts(contacts) {
   contacts.forEach(function(contact) {
     $(".contact-list").append(
       "<div class='contact'><p class='name'> " + contact + "</p></div>"
     )
   })
 }
+
+function sortByFirstName() {
```

Comment:
This is more of a style / architecture thing, but I think it might be helpful to talk about making functions "pure" by decoupling some logic here.

I would expect the function `sortByFirstName` to take some sort of list of names and return a new list sorted by first name. This particular function, however, is removing contacts from the DOM, calling `sort` on the list of contacts, and rebuilding the DOM with the new data.

In this simple example it doesn't seem to matter one way or another. But isolating the logic of `sortByFirstName` will allow it to be more reusable (should we want to use it in another context in the future, or if someone else were to hop into our code and want to use that function for some other purpose).

I would separate out the jQuery `click` events into the `document.ready` block, have them deal with any sort of DOM manipulation, and call a pure version of `sortByFirstName` directly.